Sources/SwifterFavorites.swift

```diff
@@ -34,25 +34,27 @@ public extension Swifter {
 
     If you do not provide either a user_id or screen_name to this method, it will assume you are requesting on behalf of the authenticating user. Specify one or the other for best results.
     */
-    public func getRecentlyFavouritedTweets(count: Int? = nil, sinceID: String? = nil, maxID: String? = nil, success: SuccessHandler? = nil, failure: FailureHandler? = nil) {
+    public func getRecentlyFavouritedTweets(count: Int? = nil, sinceID: String? = nil, maxID: String? = nil, tweetMode: TweetMode? = TweetMode.default, success: SuccessHandler? = nil, failure: FailureHandler? = nil) {
```

Comment:
`tweetMode` should be a non-optional field as `default` has a `nil` raw value. This applies to everywhere else in the code.